CMakeLists.txt

```diff
@@ -312,6 +312,13 @@ macro (files_hook)
     set(HDF5_FOUND OFF)
     unset(HAVE_HDF5)
   endif()
+  if(HAVE_ROCSPARSE AND HAVE_CUDA)
+    message(WARNING "WARNING! Using CUDA and HIP at the same time is not allowed. Please choose only one of them. Disabling both...\n")
+    set(CUDA_FOUND OFF)
+    set(rocsparse_FOUND OFF)
+    unset(HAVE_CUDA)
+    unset(HAVE_ROCSPARSE)
```

Comment:
This approach is very unfortunate as it is triggered by jenkins:
```
-- Performing Test HAVE_DYNAMIC_BOOST_TEST - Success
CMake Warning at CMakeLists.txt:316 (message):
  WARNING! Using CUDA and HIP at the same time is not allowed.  Please choose
  only one of them.  Disabling both...

Call Stack (most recent call first):
  mpi/install/share/opm/cmake/Modules/OpmLibMain.cmake:102 (files_hook)
  CMakeLists.txt:335 (include)
```
This results in deactivating any tests for the GPU on jenkins. I don't think this a situation we would like to be in. 
To the very least there should be one activated int this case. Even then the other will not be correctly tested automatically and that should only be a temporary solution and @akva2 would need to provide a way to select a backend on jenklns. No idea how much work that would be.

Also for users this deactivation is very surprinsing, I would be very good if the help message would include instructions on how to activate a backend.

Reply:
okay, i have disabled HIP by default and added 'hip' in trigger to (also) build with hip instead of cuda.

Reply:
Thanks for the suggestion. I updated the deactivation message showing how to switch between backends. Also, when both are used, only CUDA is deactivated. But I guess this is not that important now since Jenkins was updated. Thanks for the update!

Reply:
@akv2 what is the default jenkins build doing and how do we check cuda?